hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/TrashOzoneFileSystem.java

```diff
@@ -123,8 +126,15 @@ private void submitRequest(OzoneManagerProtocolProtos.OMRequest omRequest)
 
   @Override
   public URI getUri() {
-    throw new UnsupportedOperationException(
-        "fs.getUri() not implemented in TrashOzoneFileSystem");
+    URI uri = null;
+    try {
+      uri = new URIBuilder().setScheme(OZONE_URI_SCHEME)
+              .setHost("ofs.trash")
```

Comment:
Let's not set a hostname here & set uri to a new URI object , something like o3trash, as we are not using this anywhere and it is just to handle the exception.
```java
public static final String OZONE_O3TRASH_URI_SCHEME = "o3trash";
```

Reply:
`return Uri.create(OZONE_O3TRASH_URI_SCHEME + "://")`

Reply:
URI.create() throws exception 
   Expected authority at index 10: o3trash://
   at java.base/java.net.URI.create(URI.java:883)

Reply:
can we try with return Uri.create(OZONE_O3TRASH_URI_SCHEME + ":///")

Reply:
This works , updated and pushed new commit